Storage + Lead Guide Updates for Nicaea (#64)
Joystream/helpdesk: proposals/README.md

#### Runtime Upgrade
As before, upgrading the runtime can be proposed by any member, and voted in by the Council. This is a critical proposal that, if a "bad" runtime is proposed and voted in, can kill the blockchain.

#### Add Working Group Leader Opening

> This proposal allows an opening for a Storage Lead to be created. It requires designing an application JSON which determines what information is collected from candidates as well as determining the application and role stakes alongside various other parameters.

Reviewer: ```suggestion
This proposal allows an opening for a Storage Lead to be created. When editing the "Opening schema", you must ensure your changes still returns a valid JSON schema. This determines what information is collected from candidates. Note that the reward specified is not binding, and is only determined when the [Fill Working Group Leader Opening](#fill-working-group-leader-opening) proposal is made (and approved).
```

#### Begin Review Working Group Leader Application

This simply sets the opening for Storage Lead to the "in review" status, meaning no further applications can be accepted. It is required to move on to the `Fill Working Group Leader Opening` proposal.

#### Fill Working Group Leader Opening

> This proposal allows the council to vote on a prospective candidate.

Reviewer: ```suggestion
If the Opening is in the "Review Stage", use this proposal to propose a specific Lead. The Council can now vote, and, if approved, this will be the new Lead.

Note that there can be multiple proposals of this type at the same time, so multiple candidates can be considered simultaneously. However, once one is approved, the others will fail.
```

#### Set Working Group Mint Capacity

This effectively acts as a budget for the working group (currently referring to the Storage Working Group). The Storage Lead will be unable to spend more than the limit established by this proposal.

#### Slash Working Group Leader Stake

> To punish the Storage Lead for not performing their job correctly, they can be slashed using this proposal type.

Reviewer: ```suggestion
To punish or warn the Storage Lead for not performing their job correctly, they can be slashed partially or fully without firing them using this proposal type.
```

#### Decrease Working Group Leader Stake

> This proposal type decreases the stake of the Storage Lead.

Reviewer: ```suggestion
This proposal type allows decreasing the stake of the Storage Lead.
```

#### Set Working Group Leader Reward

> This proposal sets out the payment terms for the Storage Lead, deciding how much they should receive per interval of `n blocks`.

Reviewer: ```suggestion
This proposal allows for changing the reward for the Storage Lead if it appears too little or too much. Note that only the amount can be changed, not the frequency.
```

#### Terminate Working Group Leader Role

> If for whatever reason the Storage Lead needs to be removed from their post, this is the proposal type which needs to be voted on.

Reviewer: ```suggestion
If for whatever reason the Storage Lead needs to be removed from their post (and potentially slashed), this is the proposal type which needs to be voted on.
```
